### PR TITLE
SAK-48725 A select for assignments shouln't be visible if there are no assignment in the list

### PR DESCRIPTION
**Pull request by susowrc**
A select for assignments shouln't be visible if there are no assignment in the list

https://sakaiproject.atlassian.net/browse/SAK-48725

### CODE REVIEW

File: assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java [hunk 1]
```diff
@@ -2812,8 +2812,10 @@ private String build_list_assignments_context(VelocityPortlet portlet, Context c
         }
 
         List<Assignment> assignments = prepPage(state);
+        int allAssignmentSize = assignments.size();
```
Comment (ern) [inline assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java:2815]: It looks like there is a variable already `hasAssignments` so there is likely no need to add another?

**Comment by ern** [comment]
Thanks for the PR @susowrc

File: assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java (hunk 1 above)
Comment (susowrc) [reply assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java]: Correct. I've already changed it. Thank you so much.

**Review by jesusmmp — APPROVED** [review]

**Review by adrianfish — APPROVED** [review]
GTG. Thanks!

**Pull request merged**